script.js

```diff
@@ -1,4 +1,282 @@
+//define variables
+var others = ``;
+var shuffledCardDeck;
+var playerCard = [];
+var computerCard = [];
+var index = 0;
+var index2 = 0;
+var index4 = 0;
+index3 = 0;
+var mode = "first deal";
+var cardDeck = [];
+var playerRank = 0;
+var computerRank = 0;
+var initialMode = `first deal`;
+var currentPlayerRank = 0;
+
+// currentPlayerRank=checkRank(playerCard,playerRankArray,playerRank)
+// currentPlayerRank=checkRank(computerCard,computerRankArray,computerRank)
+// checkBust(playerRank,`player`,'computer')
+// checkBust(computerRank,`computer`,'player')
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  //make the deck
+  cardDeck = makeDeck();
+  shuffledCardDeck = shuffleCards(cardDeck);
+  if (mode == `value of ace`) {
+    if (input != `1` && input != `11`) {
+      return `Please enter either '1' or '11'`;
+    } else {
+      changingValueOfAce(input);
+    }
+  }
+  if (input == `play again`) {
```

Comment:
One thing to take note is your use of if and else if statements. You should use else if instead of if statements if only one of the conditions should happen. In your code, there are multiple if statements even though technically only one if statement should run. Even though your code still works fine. The impression given to reader might be incorrect.